### DISCUSSION TITLE
Are you familiar with Flutter ?

### DISCUSSION THREAD

**Discussion by fux0r2009** [leonlatsch/Photok#91] [Ideas]
Hi @leonlatsch I was wondering if you are familiar with Flutter ? I believe it's the future of the Mobile development (Android/iOS) So I was wondering if it would be difficult to rewrite Photok in flutter ?? Again just curious :)

**Comment by leonlatsch** [comment]
I am not famiiar with flutter.
It cloud be the future for app development, but there are some reasons for me, not to rewrite Photok in Flutter:

1. Photok is very special in the way it stores data in the interal app storage. Using Flutter for instance would mean to implement this for every platform seperateply.
2. I just think Android is the way to go for people that value privacy.
3. I just don't like Apple in general. Just like everything else, the dev account is very expensive. 

I do cross platform app dev at work but I don't think its the way I wont to go with Photok.

**Anyways** Photos is free to use, share and copy. If somebody wants Photok on another platform, he or she is free to do so, but I won't do it for now.

**Reply by fux0r2009** [reply]
I see thank you for your honest answer, and Yes I Totally share you opinion concerning apple! For now android is the way to go but in the future I am fairly certain that we will shift to a more secure Mobile OS like Fuchsia OS that google is working on... I guess until then Flutter would have matured and widely adopted by Devs :)
